server/chat-plugins/room-events.ts

```diff
@@ -222,6 +245,22 @@ export const commands: ChatCommands = {
 			this.modlog('ROOMEVENT', null, resultString);
 			return this.sendReply(resultString);
 		},
+		enable(target, room, user, cmd) {
+			if (!this.can('gamemanagement')) return false;
+			const ranks = ['%', '@', '#'];
+			const rank = target as GroupSymbol;
+			if (!ranks.includes(target)) {
+				return this.errorReply(`Invalid rank for this setting. Valid options are [%|@|#].`);
+			}
+			if (room.eventsRank === rank) return this.errorReply(`Events management is already set to ${rank}.`);
+			if (room.eventsRank !== rank && rank) {
+				room.eventsRank = rank;
+				room.chatRoomData!.eventsRank = rank;
```

Comment:
this assert isn't right, using this in a groupchat will probably crash

Reply:
does crash, resolved by blocking it in groupchats